lib/rack/tracker/google_global/google_global.rb

```diff
@@ -0,0 +1,22 @@
+class Rack::Tracker::GoogleGlobal < Rack::Tracker::Handler
+  ALLOWED_TRACKER_OPTIONS = [:cookie_domain, :user_id, 
+    :link_attribution, :allow_display_features, :anonymize_ip]
+
+  def trackers
+    options[:trackers].map do |tracker| 
+      tracker[:id].respond_to?(:call) ? tracker.merge(id: tracker[:id].call(env)) : tracker
+    end
+  end
+
+  def tracker_options(tracker)
+    return {} if tracker[:options].blank?
+    {}.tap do |options|
+      tracker[:options].slice(*ALLOWED_TRACKER_OPTIONS).each do |key, value|
+        option_value = value.respond_to?(:call) ? value.call(env) : value
+        if !option_value.nil?
+          options[key] = option_value
+        end
+      end
+    end
+  end
```

Comment:
this looks pretty universal and it could be useful to move it to the Rack::Tracker::Handler class?

Reply:
I like the idea. while I have not directy checked how the other handlers do this, we could extract allowed_tracker_options to some declarative class method? So that each tracker can configure this individually.